### ISSUE TITLE
"Animation" and "camera" conflict with https://editor.p5js.org/ defaults

### ISSUE THREAD

**Issue by waseltzer** [molleindustria/p5.play#195]
Using https://editor.p5js.org/
Added this to index.html
`<script src="https://rawgit.com/molleindustria/p5.play/master/lib/p5.play.js"></script>`
Console shows a warning:
```
p5 had problems creating the global function "Animation", possibly because your code is already using that name as a variable. You may want to rename your variable to something else. 
You just changed the value of "camera", which was a p5 function. This could cause problems later if you're not careful.
```

Example sketch: https://editor.p5js.org/wseltzer/sketches/2PvBnDHQZ

**Comment by molleindustria** [comment]
Yeah "camera" is conflicting with the webgl camera and should be renamed in p5.play although I suspect many other things would not work properly in 3D. 

I don't know about the animation though, I can't find any "Animation" reference in the p5 source code.

**Comment by quinton-ashley** [comment]
I will add a better warning for the Camera in the editor context

**Comment by quinton-ashley** [comment]
Should we rename the p5.play camera something like camera2D ? I think that'd be a huge breaking change and not beneficial since p5.play is for 2D game development and it doesn't matter that we overwrite the 3D "camera" property.

I'm thinking it would be best to just hide the warning if possible.

**Comment by quinton-ashley** [comment]
I resolved this in v2.1.2 by using p5.disableFriendlyErrors = true and setting it back to false during the document 'load' event which p5.js waits for before starting preload, setup, and draw. I think this will work in nearly every case, the user will still be able to get friendly errors afterwards.
